SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -836,28 +836,31 @@ ERPCResult USpatialSender::SendRPCInternal(UObject* TargetObject, UFunction* Fun
 
 		if (SpatialGDKSettings->UseRPCRingBuffer() && RPCService != nullptr)
 		{
-			EPushRPCResult Result = RPCService->PushRPC(TargetObjectRef.Entity, RPCInfo.Type, Payload);
+			const EPushRPCResult Result = RPCService->PushRPC(TargetObjectRef.Entity, RPCInfo.Type, Payload);
+			const EPushRPCResult Outcome = PushRPCResultUtils::GetOutcome(Result);
+			const bool bQueued = PushRPCResultUtils::GetFailureAction(Result) == EPushRPCResult::Queue;
```

Comment:
Nit: would it make a difference if it's 
`			if ((Outcome == EPushRPCResult::Success || bQueued) ||
                            (Outcome  == EPushRPCResult::Queue))
`

Reply:
Outcome does not contain the Action portion of the bitmask (where Queue is)

Reply:
I think I understand now, thanks for your replies @cmsmithio!
Now I'm wondering, why do we not have 2 enums: `EPushRPCResult ` and `EPushRPCAction` for example? I'd expect it would be easier to follow?

Reply:
Is it because you'd prefer to have 1 return variable?